### PR TITLE
Update config.lua

### PR DESCRIPTION
**Pull request by Cluuey**
Resto stuff still needs sorting.
I've probably made it longer and more complicated than optimal, lots of stance and tree settings there to make the bars keep the same order depending on the spec and affinity chosen.

The Earthwarding spell isn't working but it isn't causing an error either so I've left it how it is as, from what I read on the WoWI page, it should show something.

### CODE REVIEW

File: EventHorizon_Druid/config.lua [hunk 1]
```diff
@@ -3,227 +3,657 @@ local usemouseover = true	-- Make this false or nil (or just delete the line alt
...
+	self:newSpell({
+		cooldown = 202770,
+		stance = 4,
+		requiredTree = 1,
+		requiredTalent = 17,
+	})	
+
+	-- Feral Affinity
+	
+		-- Rake
```
Comment (Brusalk) [inline EventHorizon_Druid/config.lua]: Do you think you could fix the indentation issues that are scattered around?

File: EventHorizon_Druid/config.lua [hunk 2]
```diff
@@ -3,227 +3,657 @@ local usemouseover = true	-- Make this false or nil (or just delete the line alt
...
+	})
+
+	-- Restoration Affinity
+	
+	-- Rejuvenation
+	self:newSpell({
+		playerbuff = {774,3},
+		auraunit = target and 'mouseover' or 'target',
+        refreshable = true,
+        hasted = true,
```
Comment (Brusalk) [inline EventHorizon_Druid/config.lua]: Can this be indented to match the indentation of the other lines? Thanks!

File: EventHorizon_Druid/config.lua [hunk 3]
```diff
@@ -3,227 +3,657 @@ local usemouseover = true	-- Make this false or nil (or just delete the line alt
...
+	self:newSpell({
+		 debuff = {1822,3},
+		 refreshable = true,
+		 stance = 2,
+		 requiredTree = 2,
+		 requiredLevel = 6,
+		 requiredTalentUnselected = 17,
+	})
+
+	self:newSpell({
```
Comment (Brusalk) [inline EventHorizon_Druid/config.lua]: What is this bar for?

File: EventHorizon_Druid/config.lua [hunk 4]
```diff
@@ -3,227 +3,657 @@ local usemouseover = true	-- Make this false or nil (or just delete the line alt
...
+		playerbuff = 102558,
+		requiredTree = 3,
+		stance = {0,1,2,3,4},
+		requiredTalent = 14,
+	})
+
+	
+	-- Resto bars
+	
+--	-- Lifebloom
```
Comment (Brusalk) [inline EventHorizon_Druid/config.lua]: Are these supposed to be commented out?

File: EventHorizon_Druid/config.lua (hunk 1 above)
Comment (Cluuey) [inline EventHorizon_Druid/config.lua]: Yes, I thought I had but it had got late. I almost didn't upload anything but I was interested to see how this worked.

File: EventHorizon_Druid/config.lua (hunk 3 above)
Comment (Cluuey) [inline EventHorizon_Druid/config.lua]: I'll have a look.

Comment (Cluuey) [inline EventHorizon_Druid/config.lua]: Right, there are a couple like that. It's Rake too but dependent on having a talent selected, the OR solution you mentioned in Discord.

File: EventHorizon_Druid/config.lua [hunk 5]
```diff
@@ -3,227 +3,657 @@ local usemouseover = true	-- Make this false or nil (or just delete the line alt
...
+
+	-- Guardian bars
+	
+	-- Mangle
+	self:newSpell({
+	  unique = true,
+	  refreshable = true,
+	  hasted = true,
+	  cooldown = 33917,
+	  stance = 1,
```
Comment (Brusalk) [inline EventHorizon_Druid/config.lua]: Is it intended that every spec will have mangle?

Comment (Cluuey) [inline EventHorizon_Druid/config.lua]: No, I'll work out how that should work and update it.
There are so many spec/affinity/form combinations I missed it.

**Pull request merged**